WordPress/src/main/java/org/wordpress/android/ui/media/WordPressMediaUtils.java

```diff
@@ -260,6 +261,29 @@ public static void loadNetworkImage(String imageUrl, NetworkImageView imageView,
         }
     }
 
+    public static void loadNetworkImage(String imageUrl, WPNetworkImageView imageView) {
```

Comment:
This is almost a copy of the above `loadNetworkImage` except that it's written for a WPNetworkImageView, which seems like a maintenance problem down the road. Is there not a way to use a single routine?